Turbulence.ModelGenerator/Preprocessing.cs

```diff
@@ -62,28 +63,33 @@ public static async Task ExtractTables(Uri inPath, Uri outPath)
                     const string pattern = @"###### (.*?)( Structure| Fields)?$";
                     name = Regex.Match(line, pattern).Groups[1].Value;
                 }
-
+                
                 name = name.Replace(" ", "")
-                           .Replace("/", "")
-                           .Replace("-", "");
+                           .Replace(Path.DirectorySeparatorChar.ToString(), "")
+                           .Replace("-", "")
+                           .Replace("(", "")
+                           .Replace(")", "");
```

Comment:
might increase perf to use a single compiled regex expression here, worth a try at some point imo

Reply:
How do you make a regex that replaces all these chars in arbitrary places?

Reply:
non-capturing group & or-operator